src/report/user-report.service.ts

```diff
@@ -0,0 +1,35 @@
+import { getRepository } from 'typeorm';
+import { UserReport, CreateReportDto, UpdateStatusDto } from '.';
+
+export const ReportService = new (class {
+    async getAllReports() {
+        const reportsRepo = getRepository(UserReport);
```

Comment:
Also please refer to PR #49
There is a better and more intuitive way to access the database. We don't have to use getRepository(). We can just use all methods directly from User class.
Let me know if you need any help

this is a quick small fix

Reply:
I've tried to fix it by myself, I hope it's ok